test/junit/scala/reflect/internal/util/AbstractFileClassLoaderTest.scala

```diff
@@ -1,18 +1,14 @@
 package scala.reflect.internal.util
 
-import org.junit.Assert._
-import org.junit.Test
-import org.junit.runner.RunWith
-import org.junit.runners.JUnit4
+import org.junit.jupiter.api.Assertions._
+import org.junit.jupiter.api.Test
 
-@RunWith(classOf[JUnit4])
 class AbstractFileClassLoaderTest {
 
-  import scala.reflect.io._
-  import scala.io.Source
+  import java.net.{URL, URLClassLoader}
   import scala.io.Codec.UTF8
-  import scala.reflect.io.Streamable
-  import java.net.{ URLClassLoader, URL }
+  import scala.io.Source
+  import scala.reflect.io.{Streamable, _}
```

Comment:
this line looks like it should be just a wildcard import, but I'm not certain that there isn't something else going on